src/node.cc

```diff
@@ -3033,6 +3053,22 @@ static void ParseArgs(int* argc,
       // Also a V8 option.  Pass through as-is.
       new_v8_argv[new_v8_argc] = arg;
       new_v8_argc += 1;
+    } else if (strcmp(arg, "--report-events") == 0) {
+      const char* events = argv[index + 1];
+      if (events == nullptr) {
+        fprintf(stderr, "%s: %s requires an argument\n", argv[0], arg);
+        exit(9);
+      }
+      args_consumed += 1;
+      report_events = events;
+  fprintf(stderr, "parsed events %s \n", report_events.c_str());
+      // Replace ',' with '+' separators
+      std::size_t c = report_events.find_first_of(",");
+      while (c != std::string::npos) {
+        report_events.replace(c, 1, "+");
+        c = report_events.find_first_of(",", c + 1);
+      }
+  fprintf(stderr, "filtered events %s \n", report_events.c_str());
```

Comment:
can we have a default value for `--report-events` ? i.e., in case if user specifies this flag but no values, how about picking the most relevant trace events and feed those into `report_events` filed?